File: 32481163 - EditingCrashExample/EditingCrashExample/TableViewController.m
//
//  TableViewController.m
//  EditingCrashExample
//
//  Created by Konstantin Bender on 5/29/17.
//  Copyright © 2017 PSPDFKit. All rights reserved.
//

#import "TableViewController.h"

@interface TableViewController ()

@end

@implementation TableViewController

- (void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear:animated];

    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Radar" message:@"Tap Edit\nTap in one of the text fields to begin editing\nTap Done" preferredStyle:UIAlertControllerStyleAlert];
    [alert addAction:[UIAlertAction actionWithTitle:@"Oh damn, another 💥" style:UIAlertActionStyleDefault handler:nil]];
    [self presentViewController:alert animated:YES completion:NULL];
}

- (void)viewDidLoad {
    [super viewDidLoad];
    self.navigationItem.rightBarButtonItem = self.editButtonItem;
}

#pragma mark - Table view data source

- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
    return 1;
}

- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
    return 5;
}

- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
    return [tableView dequeueReusableCellWithIdentifier:@"test" forIndexPath:indexPath];
}

@end

File: FB7515804 - Popovers which are shown from a button in an inputAccessoryView have extra space between the button and the arrow/PopoverPlayground/TableViewController.swift
//
//  TableViewController.swift
//  PopoverPlayground
//
//  Created by Akshat Patel on 12/24/19.
//  Copyright © 2019 akshat. All rights reserved.
//

import UIKit

class TableViewController: UITableViewController {

    // MARK: - Table view data source

    override func numberOfSections(in tableView: UITableView) -> Int {
        return 1
    }

    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return 30
    }

    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let cell = tableView.dequeueReusableCell(withIdentifier: "reuseIdentifier", for: indexPath)
        cell.textLabel?.text = "\(indexPath.row + 1)"
        return cell
    }
}
